sdks/java/io/kafka/build.gradle

```diff
@@ -107,6 +107,9 @@ dependencies {
   testImplementation library.java.testcontainers_kafka
   testRuntimeOnly library.java.slf4j_jdk14
   testRuntimeOnly project(path: ":runners:direct-java", configuration: "shadow")
+  testImplementation project(path: ":runners:google-cloud-dataflow-java")
```

Comment:
Consider a separate subproject. There is a potential circular dependency such that `:runners:google-cloud-dataflow-java` has compile time dependency on KafkaIO